docs/en/core-services/Caching.md

```diff
@@ -204,70 +190,99 @@ A typical configuration will have three tiers for Memory, SSD, and HDD.
 To use multiple hard drives in the HDD tier, specify multiple paths when configuring
 `alluxio.worker.tieredstore.level{x}.dirs.path`.
 
-## Evicting Stale Data
-
-When Alluxio storage is full, Alluxio frees up space for new data as its storage is designed to be
-volatile. Eviction is the mechanism that removes old data.
-
-Alluxio applies asynchronous eviction in the background. It relies on a periodic space
-reserver thread in each worker to evict data. When the worker storage utilization reaches a
-maximum threshold, it evicts data until the usage reaches a minimum threshold. The two thresholds
-are configurable, labeled as the high and low watermarks, respectively. In the case where we have
-216 GB of storage on a worker, eviction can be triggered at about 200 GB to reduce storage
-utilization to 160 GB by setting the high watermark to 90% and the low watermark to 75%.
-
-```properties
-alluxio.worker.tieredstore.level0.watermark.high.ratio=0.9
-alluxio.worker.tieredstore.level0.watermark.low.ratio=0.75
-```
-
-Asynchronous eviction is particularly useful for write or read-cache heavy workloads compared to
-synchronous eviction used prior to Alluxio 2.0, which waits for a client to request more space than
-is currently available on the worker and then kicks off the eviction process to free up enough
-space to serve that request. This
-leads to many small eviction attempts, which is less efficient, but maximizes the utilization of
-available Alluxio space.
+### Block Annotation Policies
 
-### Eviction Policies
+Alluxio uses block annotation policies, starting v2.3, to maintain strict ordering of blocks in storage. 
+Annotation policy defines an order for blocks across tiers and is consulted during:
+- Eviction
+- Dynamic Block Placement
 
-Alluxio uses evictors to decide which blocks to remove from Alluxio storage.
-Users can specify different Alluxio evictors to control the eviction process better.
+The eviction, that happens in-line with writes, will attempt to remove blocks based on the order enforced by the block annotation policy.
+The last block in annotated order is the first candidate for eviction regardless of which tier it's sitting on.
```

Comment:
Previously eviction happens on reaching high watermark, and executed by a background process. Does it mean that the foreground writer thread will pause and evict blocks synchronously?

Reply:
Yes.

Reply:
Added more explanation to `Writing Data` section.